3scale-qe/3scale-tests
# Another batch of backports

* Update field in context from openshift-client-python (api_url -> api_server)
* Fix selection of rhsso in rhaom
* Save pytest_metadata as json file
* Add Prometheus availability test
* Remove unused tools
* Fix testsuite crash when tool is not available
* Add tool check for tools with route
* Add httpbin availability check
* Update to use offset-aware time objects
* Add option --sso-only to deselect all tests except rhsso/rhbk
* Fixed and stabilized billing tests
